src/styles/blocks/price-number.scss

```diff
@@ -0,0 +1,9 @@
+.price-number {
```

Comment:
you don't need create file for each classname, first use bem, and parent selector
https://mate-academy.github.io/fe-program/css/typical-bem-mistakes
https://sass-lang.com/documentation/style-rules/parent-selector/